src/main/java/seedu/address/logic/parser/FilledCommandArgument.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.logic.parser;
+
+public class FilledCommandArgument extends CommandArgument {
+    protected FilledCommandArgument(Prefix prefix, String value) {
+        super(value, prefix, false, false);
```

Comment:
Personally kinda weird that the order for the parameter functions got swapped from the constructor to the `super()`. Up to personal preference I guess

Reply:
true